src/IO/Resources/AnimationsLoader.cs

```diff
@@ -1385,6 +1385,8 @@ public bool LoadAnimationFrames(ushort animID, byte animGroup, byte direction, r
             return true;
         }
 
+        [ThreadStatic] private static byte[] _decompressedData = new byte[0x1000];
```

Comment:
`[ThreadStatic]` causes to create 2 instances of `_decompressedData`:
1 at loading time due to the `Task.Run`
1 from the main thread